backend/modules/student/routers/student_instances.py

```
@@ -241,13 +241,30 @@ async def get_student_simulation_instances(
     """Get all simulation instances for the current student"""
+    import json as json_lib
+    endpoint_start = time.time()
     try:
         from common.services.cache_service import redis_manager
         
-        # Check Redis cache first (short TTL for dashboard data freshness)
+        # #region agent log
+        try:
+            with open('/Users/amybihag/n-aible_edtech_sims/.cursor/debug.log', 'a') as f:
+                f.write(json_lib.dumps({"sessionId":"debug-session","runId":"run1","hypothesisId":"A","location":"student_instances.py:get_student_simulation_instances","message":"Endpoint called","data":{"user_id":current_user.id,"status_filter":status_filter,"cohort_id":cohort_id},"timestamp":int(time.time()*1000)})+"\n")
+        except: pass
```

Ruff (0.14.8)

Do not use bare `except`

(E722)